src/main/java/com/couchbase/lite/replicator/ChangeTracker.java

```diff
@@ -569,4 +582,24 @@ public void setUsePOST(boolean usePOST) {
 
     }
 
+
+    public void setPaused(boolean paused) {
+        Log.v(Log.TAG, "setPaused: " + paused);
+        synchronized (pausedObj) {
+            this.paused = paused;
+            pausedObj.notifyAll();
+        }
+    }
+
+    protected void waitIfPaused(){
```

Comment:
I couldn't find the long poll mode doing this. How does the long poll mode handle pause?